components/automate-cli/cmd/chef-automate/ha_node_delete_aws.go

```diff
@@ -139,6 +143,17 @@ func (dna *DeleteNodeAWSImpl) modifyConfig() error {
 }
 
 func (dna *DeleteNodeAWSImpl) prepare() error {
+	// Stop all services on the node to be deleted
+	infra, _, err := dna.nodeUtils.getHaInfraDetails()
+	if err != nil {
+		return err
+	}
+
+	err = dna.nodeUtils.stopServicesOnNode(dna.automateIpList, dna.chefServerIpList, dna.postgresqlIpList, dna.opensearchIpList, infra, dna.sshUtil)
+	if err != nil {
+		return status.Wrap(err, status.CommandExecutionError, "Error stoping services on node")
+	}
+
```

Comment:
run this after terraform has finished removal of that node

Reply:
done